instagram-app/src/components/PostContainer/PostsPage.jsx

```diff
@@ -49,6 +49,7 @@ export default class PostsPage extends Component {
       }
     
       render(){
+        console.log('loading page');
         return (
           <div className="App container">

```

Comment:
I would have liked to have seen the username stored somewhere in state on login, this way you can dynamically add the logged in users username as the preface to their own comments on add.